OCAT/client/pages/Assessments/NewAssessment.jsx

```diff
@@ -1,16 +1,227 @@
+
 import React from 'react';
 import { Button, Form } from 'react-bootstrap';
+import { useForm } from 'react-hook-form';
 import { AssessmentService } from '../../services/AssessmentService';
 
 export const NewAssessment = () => {
+  const { formState: { errors }, handleSubmit, register } = useForm({
+    defaultValues: {
+      altercations_with_cats: ``,
+      altercations_with_owner: ``,
+      cat_name: ` `,
+      cat_date_of_birth: ``,
+      hisses_at_strangers: ``,
+      plays_with_dogs: ``,
+      previous_contact: ``,
+    },
+
+  });
 
   // create a form that utilizes the "onSubmit" function to send data to OCAT/client/services/AssessmentService.js and
   // then onto the OCAT/server/routes/AssessmentAPI express API
+
   const onSubmit = async (data) => {
-    await AssessmentService.submit(data);
+    try
+    {
+      const num1 = parseInt(data.previous_contact);
+      const num2 = parseInt(data.altercations_with_cats);
+      const num3 = parseInt(data.altercations_with_owner);
+      const num4 = parseInt(data.plays_with_dogs);
+      const num5 = parseInt(data.hisses_at_strangers);
+      let risk_level = ``;
+      const created_at = new Date();
+
+      const score = num1 + num2 + num3 + num4 + num5;
+      console.log(num1);
+      console.log(num2);
+      console.log(num3);
+      console.log(num4);
+      console.log(num5);
+      console.log(`score:${score}`);
+
+      switch (score) {
+        case 2:
+        case 3:
+          risk_level = `Medium Risk`;
+          break;
+        case 4:
+        case 5:
+          risk_level = `High Risk`;
+          break;
+        default:
+          risk_level = `low Risk`;
+      }
+      console.log(risk_level);
+      console.log(created_at);
+      console.log(`data`, data);
+
+      await AssessmentService.submit({ score, risk_level, created_at, data });
+    }
+    catch (error) {
+      throw new Error(`${error.response.statusText} - ${error.response.data.message}`);
+    }
   };
 
-  return <Form>
+  console.log(errors);
+  return <Form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
I'm not going to go through this and fix everything, but your formatting here is pretty all over the place.  lint rules will fix some of this, but it's better to be consistent with your formatting.